src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Aggregator/FunctionResultLog.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host.Loggers
+{
+    internal class FunctionResultLog
```

Comment:
I'm not sure why I didn't do that. I can add `Duration` and `Success` (or even figure out `Success` by looking at `ErrorDetails`) and just get rid of `FunctionResultLog`. I'll do that.